api/feature_latency_api.py

```diff
@@ -0,0 +1,140 @@
+# Copyright 2024 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License")
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from datetime import datetime, timedelta
+import logging
+
+from chromestatus_openapi.models.feature_link import FeatureLink
+from chromestatus_openapi.models.feature_latency import FeatureLatency
+
+from api import channels_api
+from framework import basehandlers
+from internals import stage_helpers
+from internals.core_enums import *
+from internals.core_models import FeatureEntry, Stage
+
+
+class FeatureLatencyAPI(basehandlers.APIHandler):
+  """Implements the OpenAPI /feature_latency path."""
+
+  def get_date_range(
+      self, request_args: dict[str, str]
+  ) -> tuple[datetime|None, datetime|None]:
+    """Parse start and end dates from query-string params."""
+    start_param: str | None = request_args.get('startDate', None)
```

Comment:
I think of the python code using snake case, and the JS code using camel case, and the names of parameters following the convention of the code that sent the parameter.  It can look a little out of place when you have a camel case name appearing in a python file, but that tells you that it came from JS.